opensearch-operator/pkg/builders/dashboards.go

```diff
@@ -65,22 +69,31 @@ func NewDashboardsDeploymentForCR(cr *opsterv1.OpenSearchCluster) *sts.Deploymen
 									ContainerPort: port,
 								},
 							},
+							StartupProbe:  &probe,
+							LivenessProbe: &probe,
 							Env: []corev1.EnvVar{
 								{
-									Name:      "OPENSEARCH_HOSTS",
-									Value:     "https://" + cr.Spec.General.ServiceName + "-svc" + "." + cr.Spec.General.ClusterName + ":9200",
-									ValueFrom: nil,
+									Name:  "OPENSEARCH_HOSTS",
+									Value: "https://" + cr.Spec.General.ServiceName + "." + cr.Spec.General.ClusterName + ".svc.cluster.local:9200",
```

Comment:
also url is configured. please take from configuration

Reply:
fixed